deploy-to-aks/README.md

```diff
@@ -0,0 +1,41 @@
+# Deploy to AKS
+
+Deploy a docker image by sha to a specific service environment.
+Optionally after deployment
+- run a seed command for review apps
+  - if enabled, it will run 'make review seed-review-app' so this must exist in the Makefile
+- run a smoktest after deployment
+
+If using Google Cloud then GCP_PROJECT_ID abd GCP_WIP variables must be set in the service Makefile.
+
+## Inputs
+- `azure-credentials`: A JSON string containing service principle credentials (Required)
+- `gcp-wip`: The full identifier of the GCP Workload Identity Provider (Optional)
+- `environment`: Name of the environment to deploy (Required)
+- `github-token`: Default Github token retrieved via secrets. GITHUB_TOKEN or PAT with permission to the repository (Required)
+- `pr-number`: Pull Request Number if deploying a review app (Optional)
+- `db-seed`: Run seed command after a deployment. Should only be used for review apps (default: false)
+- `sha`: commit sha of the docker image to be deployed (Required)
+- `slack-webhook` : A slack webhook to send a slack message to the service tech channel on deploy failure (Optional)
+- `smoke-test` : Run an application smoke test after deployment (default: false)
+- `healthcheck` : Health check path (Optional)
+- `terraform-base` : Name of the base terraform path (default: 'terraform/application')
+
+## Example
+
+```yaml
+- name: Checkout
```

Comment:
Could we include checkout in the action?

Reply:
moved to action